content/articles/react-components-instantiation/index.md

```diff
@@ -0,0 +1,188 @@
+### React Components  Instantiation
+A React Component is a small, reusable bit of code responsible for rendering HTML. In React, we can define a component as a class or function. 
+
+For a React component to be defined, we will create a component and then extend the `React.component class`. For example, to create a classroom component, the classroom component should have the classroom details.
+
+Throughout this tutorial, we will be mostly talking about JSX. The JSX element can be written in HTML format or written as a component instance. In JSX, you use capitalization to differentiate between HTML-like and component instances.
+
+#### Prerequisites
+To effectively understand the tutorial, the reader will need the following:
+- Basic understanding of the React library
+- A suitable IDE such as VS Code.
+- Basic understanding of JSX
+
+**In this tutorial, we will use [JS complete](https://jscomplete.com/playground).To use JS complete with React, first install the [React Developer Tools extension.](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)**
+
+#### Key takeaways 
+- What is component instantiation?
+- How to instantiate a React component 
+- Difference between HTML like  and Component instances in JSX elements
+- New keyword meaning and how it is used in React
+- React Component  Instantiation
+
+#### Instantiating the React component
+Let's begin by importing the React library.
+
+```js
+  import React from 'react'; 
+```
+
+We are importing React because JSX will be converted to regular JavaScript, which uses React's `React.createElement` method. Therefore, we need to import React when using JSX. Your normal compiler does not understand JSX.This is like telling your compiler you provide a JSX file.
+
+After importing the React library, we will import ReactDOM. ReactDom is a different module. If you are not using ES6, install `react-dom` in your IDE using `npm install react-dom`. But since we will be using ES6, we will import `ReactDOM` using the code below:
+
+```js
+    import ReactDOM from 'react-dom';
+```    
+ReactDOM is important for rendering JSX elements and react components to the DOM. The DOM is the representation of the whole user interface of your application. Let's now create a component class. To create our component class, we will subclass React.Component. We will do this by using the following syntax: `class V extends React.Component {}`. This is how ES6 creates a stateful component. 
+
+We will now add the render function to our code:
+
+```js
+   class V extends React.Component {
+   render() {}
+   }
+```
+The motive of the render function in our application is to enable React to render HTML on our webpage. It's a must for a render function to have a return statement. A JSX expression is returned by the return statement in the render function as shown:
```

Comment:
Every moment you use a function, ensure it's differentiated from your sentences, as I had suggested above, in this case, have `render()`. Fix throughout your article!